e2e/cucumber-features/simple.feature

```diff
@@ -12,4 +12,4 @@ Background:
 Scenario: compose up
     When I run "compose up -d"
     Then the output contains "simple-1  Started"
-    And service "simple" is "running"
+    And service "simple" is "Up Less than a second"
```

Comment:
shall we avoid depending on time since created output here, and only consider `Up`?

Reply:
yes! I meant to revisit this before pushing, and then I forgot. That's a recipe for flaky tests 😅